docs/sdk/serverless-signing/combining-signatures.md

```diff
@@ -1,102 +1,74 @@
+When [signing a transaction](../serverless-signing/quick-start#sign-a-transaction.md) with Lit, signature shares are typically combined client-side. However, the `signAndCombineEcdsa` function allows you to combine signature shares directly within a Lit Action, which is useful for when you want to make use of the signed data within your Lit Action e.g. submitting a signed transaction. The signature shares will remain within the confines of each Lit node's [Trusted Execution Environment (TEE)](../../resources/how-it-works#1-lit-nodes.md) without ever being exposed to the outside world. 
 
-When you call the `signAndCombineEcdsa` function, signature shares are collected from each Lit node before being combined on a *single* node. The following code example will show how you can use this functionality for arbitrary message signing, as well as how to sign a blockchain transaction using ethers.js. A complete code example is linked at the bottom of this page. 
+When you call the `signAndCombineEcdsa` function, signature shares are collected from each Lit node before being combined on a *single* node. The following code example will show how you can use this functionality for signing a blockchain transaction using ethers.js.
 
-## Signing a message
+## Prerequisites
 
-The following Lit Action uses `signAndCombineEcdsa` to sign the message 'hello world'. 
+- Knowlege of [SessionSigs](../authentication/session-sigs/intro)
+- Basic understanding of [Lit Actions](../serverless-signing/quick-start)
 
-```js
-const code = `(async () => {
-  // sign "hello world" and allow all the nodes to combine the signature and return it to the action.
-  const utf8Encode = new TextEncoder();
-  const toSign = utf8Encode.encode('Hello World');
-  ethers.utils.arrayify(
-    ethers.utils.keccak256(toSign)
-  );
-  // Will use the authentication provided to the "executeJs" call from the sdk on the client.
+## Complete Code Example
+
+The complete code example is available in the [Lit Developer Guides Code Repository](https://github.com/LIT-Protocol/developer-guides-code/tree/master/sign-and-combine-ecdsa/nodejs). There you can find a Node.js implementation of the code.
+
+### Example Lit Action
+
+The following Lit Action uses `signAndCombineEcdsa` to combine the signatures created by each share signing the `toSign` variable, which is the provided hash of a serialized transaction. We then serialize the transaction and sign it using `ethers.js`.
```

Comment:
> The following Lit Action uses `signAndCombineEcdsa` to combine the signatures created by each share signing the `toSign` variable, which is the provided hash of a serialized transaction. We then serialize the transaction and sign it using `ethers.js`.

Imo this sentence is a little confusing. Is the transaction already serialized, and then we serialize again? Or are these two separate steps?

Also , i think we could make it more clear that each node is provisioning a share of the key which must be combined to form the complete signature

Reply:
Changed to:

The following Lit Action uses `signAndCombineEcdsa` to combine partial signatures provided by each node, each of which signs a share of the `toSign` variable—this variable contains the hash of the serialized transaction. After combining these shares into a complete signature, we use `ethers.js` to serialize the transaction again, this time including the signature, to finalize it for submission.

Reply:
sounds great